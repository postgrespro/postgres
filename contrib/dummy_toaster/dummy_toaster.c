/*-------------------------------------------------------------------------
 *
 * dummy_toaster.c
 *		Bloom index utilities.
 *
 * Portions Copyright (c) 2016-2021, PostgreSQL Global Development Group
 * Portions Copyright (c) 1990-1993, Regents of the University of California
 *
 * IDENTIFICATION
 *	  contrib/bloom/blutils.c
 *
 *-------------------------------------------------------------------------
 */
#include "postgres.h"
#include "fmgr.h"

PG_MODULE_MAGIC;
PG_FUNCTION_INFO_V1(dummy_toaster_handler);

Datum
dummy_toaster_handler(PG_FUNCTION_ARGS)
{
	/*
	IndexAmRoutine *amroutine = makeNode(IndexAmRoutine);

	amroutine->amstrategies = BLOOM_NSTRATEGIES;
	*/

	elog(ERROR, "dummy_toaster_handler is uninimplemented yet");
	PG_RETURN_POINTER(NULL);
}

