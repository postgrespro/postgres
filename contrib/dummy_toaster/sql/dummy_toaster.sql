CREATE EXTENSION dummy_toaster;

CREATE TABLE tst_failed (
	t text TOASTER dummy_toaster TOASTER dummy_toaster
);

CREATE TABLE tst1 (
	t text TOASTER dummy_toaster
);

CREATE TABLE tst2 (
	t	text
);

ALTER TABLE tst2 ALTER COLUMN t SET TOASTER dummy_toaster;


