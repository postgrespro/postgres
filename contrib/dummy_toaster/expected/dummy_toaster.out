CREATE EXTENSION dummy_toaster;
CREATE TABLE tst_failed (
	t text TOASTER dummy_toaster TOASTER dummy_toaster
);
ERROR:  multiple TOASTER clauses not allowed
CREATE TABLE tst1 (
	t text TOASTER dummy_toaster
);
ERROR:  unimplemented yet (toaster: dummy_toaster)
CREATE TABLE tst2 (
	t	text
);
ALTER TABLE tst2 ALTER COLUMN t SET TOASTER dummy_toaster;
ERROR:  unimplemented yet (toaster: dummy_toaster)
